src/pe_source/data/sixgill/source.py

```diff
@@ -0,0 +1,140 @@
+"""Scripts for importing Sixgill data into PE Postgres database."""
+
+# Standard Python Libraries
+import logging
+
+# Third-Party Libraries
+import pandas as pd
+import requests
+
+from .api import (
+    alerts_count,
+    alerts_list,
+    credential_auth,
+    dve_top_cves,
+    intel_post,
+    org_assets,
+)
+
+
+def alias_organization(org_id):
+    """List an organization's aliases."""
+    assets = org_assets(org_id)
+    df_assets = pd.DataFrame(assets)
+    aliases = df_assets["organization_aliases"].loc["explicit":].tolist()[0]
+    return aliases
+
+
+def root_domains(org_id):
+    """Get root domains."""
+    assets = org_assets(org_id)
+    df_assets = pd.DataFrame(assets)
+    root_domains = df_assets["domain_names"].loc["explicit":].tolist()[0]
+    return root_domains
+
+
+def mentions(date, aliases):
+    """Pull dark web mentions data for an organization."""
+    mentions = ""
+    for mention in aliases:
+        mentions += '"' + mention + '"' + ","
+    mentions = mentions[:-1]
+    query = "site:forum_* AND date:" + date + " AND " + "(" + str(mentions) + ")"
+    logging.info("Query:")
+    logging.info(query)
+    count = 0
+    while count <= 5:
+        try:
+            logging.info("Intel post try #%s", count + 1)
+            resp = intel_post(query, frm=0, scroll=False, result_size=1)
+            count = 6
+        except Exception:
+            logging.info("Error. Trying intel_post again...")
+            count += 1
+            continue
+    count_total = resp["total_intel_items"]
+    logging.info("Total Mentions: %s", count_total)
+
+    i = 0
+    all_mentions = []
+    if count_total < 10000:
+        while i < count_total:
+            # Recommended "from" and "result_size" is 50. The maximum is 400.
+            resp = intel_post(query, frm=i, scroll=False, result_size=200)
+            i = i + 200
+            logging.info("Getting %s of %s....", i, count_total)
+            intel_items = resp["intel_items"]
+            df_mentions = pd.DataFrame.from_dict(intel_items)
+            all_mentions.append(df_mentions)
+            df_all_mentions = pd.concat(all_mentions).reset_index(drop=True)
+
+    else:
+        while i < count_total:
+            # Recommended "from" and "result_size" is 50. The maximum is 400.
+            resp = intel_post(query, frm=i, scroll=True, result_size=400)
+            i = i + 400
```

Comment:
More common Pythonic way of writing this:
```suggestion
            i += 400
```